backend/src/api/blocks.ts

```diff
@@ -462,6 +463,9 @@ class Blocks {
           }
           await blocksRepository.$saveBlockInDatabase(blockExtended);
 
+          // Currently Commented it as during every main loop update block headers are already syncing
+          // await opBlockHeaderService.$saveBlockHeader(this.currentBlockHeight);
```

Comment:
1. pushing commented code is not a good idea: if it shouldn't be running, then it shouldn't be here, as for me;
2. opBlockHeaderService.$saveBlockHeader() is not total, meaning, that it will only persist `argument - 6` block: 
$syncOlderBlockHeader() is able to persist [last_persisted_block;current_tip - 8] which can handle case of, say restarting of bitcoin node after hour of being offline or due to out of space event... so $syncOlderBlockHeader() is a better candidate as an entry point to persistence layer

Reply:
Right, syncOlderBlockHeader is the only function which will be expose outside the class and at persistence layer